resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_0.2_abs_high.inst.cfg

```diff
@@ -0,0 +1,16 @@
+[general]
+version = 4
+name = High
+definition = tizyx_evy
+
+[metadata]
+setting_version = 5
```

Comment:
Currently we are already using `setting_version = 6`. You can use find and replace.